### ISSUE TITLE
increase color contrast on category detail links

### ISSUE THREAD

**Issue by skorasaurus** [opencleveland/refundcleveland#52]
The contrast for the "see details" links should be increased to meet accessibility (WCAG 2.1) requirements. 

a.category_detail

**Comment by skorasaurus** [comment]
fixed in https://github.com/opencleveland/refundcleveland/pull/95
